lib/jsdom/living/register-elements.js

```diff
@@ -355,6 +355,8 @@ const mappings = {
   }
 };
 
+// N.B. the "core" here is the "extraDOM" object imported from living/index.js, not the core tied to a specific Window.
+// Thus, it only contains the limited set of objects in living/index.js, but that is enough currently.
 module.exports = core => {
```

Comment:
Let's just update the parameter name then, instead of creating a comment explaining why it's misleading.